[Bug]: crash when importing aws_datazone_project

### Terraform Core Version

1.9.3

### AWS Provider Version

5.63

### Affected Resource(s)

aws_datazone_project

When importing the provider crashed (see below)

import_aws_datazone_project.tf:

import {
  to = aws_datazone_project.cdsvp6st8l82lj
  id = "cdsvp6st8l82lj"
}


$ terraform plan -generate-config-out=resources.out


Stack trace from the terraform-provider-aws_v5.63.0_x5 plugin:

panic: runtime error: index out of range [1] with length 1

goroutine 81 [running]:
github.com/hashicorp/terraform-provider-aws/internal/service/datazone.(*resourceProject).ImportState(0x1161aea28?, {0x1161aea28, 0x140021716b0}, {{0x1400186f230?, 0x0?}, {0x0?}}, 0x14001226180)
	github.com/hashicorp/terraform-provider-aws/internal/service/datazone/project.go:323 +0x278
github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider.(*wrappedResource).ImportState(0x14002dbc780?, {0x1161aea28?, 0x140009eb9e0?}, {{0x1400186f230?, 0x110fbf96a?}, {0x2d?}}, 0x14001226180)
	github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider/intercept.go:329 +0xb4
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ImportResourceState(0x14000fce960, {0x1161aea28, 0x140009eb9e0}, 0x14000fe9730, 0x140021d95d0)
	github.com/hashicorp/terraform-plugin-framework@v1.11.0/internal/fwserver/server_importresourcestate.go:136 +0x678
github.com/hashicorp/terraform-plugin-framework/internal/proto5server.(*Server).ImportResourceState(0x14000fce960, {0x1161aea28?, 0x140009eb8f0?}, 0x14000abc9f0)
	github.com/hashicorp/terraform-plugin-framework@v1.11.0/internal/proto5server/server_importresourcestate.go:47 +0x248
github.com/hashicorp/terraform-plugin-mux/tf5muxserver.(*muxServer).ImportResourceState(0x14000f9ba40, {0x1161aea28?, 0x14000abca50?}, 0x14000abc9f0)
	github.com/hashicorp/terraform-plugin-mux@v0.16.0/tf5muxserver/mux_server_ImportResourceState.go:36 +0x184
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ImportResourceState(0x140010ca320, {0x1161aea28?, 0x14000a51ec0?}, 0x14000d7c190)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:896 +0x1e8
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ImportResourceState_Handler({0x115cddd20, 0x140010ca320}, {0x1161aea28, 0x14000a51ec0}, 0x14004cbe000, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:536 +0x1c0
google.golang.org/grpc.(*Server).processUnaryRPC(0x14000f2e200, {0x1161aea28, 0x14000a50ae0}, {0x116219aa0, 0x140002aa300}, 0x140009b8000, 0x1400214b350, 0x11f92f010, 0x0)
	google.golang.org/grpc@v1.63.2/server.go:1369 +0xb58
google.golang.org/grpc.(*Server).handleStream(0x14000f2e200, {0x116219aa0, 0x140002aa300}, 0x140009b8000)
	google.golang.org/grpc@v1.63.2/server.go:1780 +0xb20
google.golang.org/grpc.(*Server).serveStreams.func2.1()
	google.golang.org/grpc@v1.63.2/server.go:1019 +0x8c
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 43
	google.golang.org/grpc@v1.63.2/server.go:1030 +0x13c
panic: runtime error: index out of range [1] with length 1

goroutine 84 [running]:
github.com/hashicorp/terraform-provider-aws/internal/service/datazone.(*resourceProject).ImportState(0x1161aea28?, {0x1161aea28, 0x1400224a600}, {{0x1400186fca0?, 0x0?}, {0x0?}}, 0x14001e004e0)
	github.com/hashicorp/terraform-provider-aws/internal/service/datazone/project.go:323 +0x278
github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider.(*wrappedResource).ImportState(0x14004c78380?, {0x1161aea28?, 0x14001174d80?}, {{0x1400186fca0?, 0x110fbf96a?}, {0x2d?}}, 0x14001e004e0)
	github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider/intercept.go:329 +0xb4
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ImportResourceState(0x14000fce960, {0x1161aea28, 0x14001174d80}, 0x14001025ea0, 0x140021e95d0)
	github.com/hashicorp/terraform-plugin-framework@v1.11.0/internal/fwserver/server_importresourcestate.go:136 +0x678
github.com/hashicorp/terraform-plugin-framework/internal/proto5server.(*Server).ImportResourceState(0x14000fce960, {0x1161aea28?, 0x14001174c90?}, 0x14000daea50)
	github.com/hashicorp/terraform-plugin-framework@v1.11.0/internal/proto5server/server_importresourcestate.go:47 +0x248
github.com/hashicorp/terraform-plugin-mux/tf5muxserver.(*muxServer).ImportResourceState(0x14000f9ba40, {0x1161aea28?, 0x14000daeab0?}, 0x14000daea50)
	github.com/hashicorp/terraform-plugin-mux@v0.16.0/tf5muxserver/mux_server_ImportResourceState.go:36 +0x184
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ImportResourceState(0x140010ca320, {0x1161aea28?, 0x14000abd020?}, 0x14000d7cbe0)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:896 +0x1e8
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ImportResourceState_Handler({0x115cddd20, 0x140010ca320}, {0x1161aea28, 0x14000abd020}, 0x14004cbe780, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:536 +0x1c0
google.golang.org/grpc.(*Server).processUnaryRPC(0x14000f2e200, {0x1161aea28, 0x14000abcf60}, {0x116219aa0, 0x140002aa300}, 0x14000a44c60, 0x1400214b350, 0x11f92f010, 0x0)
	google.golang.org/grpc@v1.63.2/server.go:1369 +0xb58
google.golang.org/grpc.(*Server).handleStream(0x14000f2e200, {0x116219aa0, 0x140002aa300}, 0x14000a44c60)
	google.golang.org/grpc@v1.63.2/server.go:1780 +0xb20
google.golang.org/grpc.(*Server).serveStreams.func2.1()
	google.golang.org/grpc@v1.63.2/server.go:1019 +0x8c
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 43
	google.golang.org/grpc@v1.63.2/server.go:1030 +0x13c

Error: The terraform-provider-aws_v5.63.0_x5 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.




### Expected Behavior

Import the resource without a crash



### Actual Behavior

Provider crashes - see below

### Relevant Error/Panic Output Snippet

```shell
Stack trace from the terraform-provider-aws_v5.63.0_x5 plugin:

panic: runtime error: index out of range [1] with length 1

goroutine 81 [running]:
github.com/hashicorp/terraform-provider-aws/internal/service/datazone.(*resourceProject).ImportState(0x1161aea28?, {0x1161aea28, 0x140021716b0}, {{0x1400186f230?, 0x0?}, {0x0?}}, 0x14001226180)
	github.com/hashicorp/terraform-provider-aws/internal/service/datazone/project.go:323 +0x278
github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider.(*wrappedResource).ImportState(0x14002dbc780?, {0x1161aea28?, 0x140009eb9e0?}, {{0x1400186f230?, 0x110fbf96a?}, {0x2d?}}, 0x14001226180)
	github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider/intercept.go:329 +0xb4
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ImportResourceState(0x14000fce960, {0x1161aea28, 0x140009eb9e0}, 0x14000fe9730, 0x140021d95d0)
	github.com/hashicorp/terraform-plugin-framework@v1.11.0/internal/fwserver/server_importresourcestate.go:136 +0x678
github.com/hashicorp/terraform-plugin-framework/internal/proto5server.(*Server).ImportResourceState(0x14000fce960, {0x1161aea28?, 0x140009eb8f0?}, 0x14000abc9f0)
	github.com/hashicorp/terraform-plugin-framework@v1.11.0/internal/proto5server/server_importresourcestate.go:47 +0x248
github.com/hashicorp/terraform-plugin-mux/tf5muxserver.(*muxServer).ImportResourceState(0x14000f9ba40, {0x1161aea28?, 0x14000abca50?}, 0x14000abc9f0)
	github.com/hashicorp/terraform-plugin-mux@v0.16.0/tf5muxserver/mux_server_ImportResourceState.go:36 +0x184
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ImportResourceState(0x140010ca320, {0x1161aea28?, 0x14000a51ec0?}, 0x14000d7c190)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:896 +0x1e8
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ImportResourceState_Handler({0x115cddd20, 0x140010ca320}, {0x1161aea28, 0x14000a51ec0}, 0x14004cbe000, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:536 +0x1c0
google.golang.org/grpc.(*Server).processUnaryRPC(0x14000f2e200, {0x1161aea28, 0x14000a50ae0}, {0x116219aa0, 0x140002aa300}, 0x140009b8000, 0x1400214b350, 0x11f92f010, 0x0)
	google.golang.org/grpc@v1.63.2/server.go:1369 +0xb58
google.golang.org/grpc.(*Server).handleStream(0x14000f2e200, {0x116219aa0, 0x140002aa300}, 0x140009b8000)
	google.golang.org/grpc@v1.63.2/server.go:1780 +0xb20
google.golang.org/grpc.(*Server).serveStreams.func2.1()
	google.golang.org/grpc@v1.63.2/server.go:1019 +0x8c
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 43
	google.golang.org/grpc@v1.63.2/server.go:1030 +0x13c
panic: runtime error: index out of range [1] with length 1

goroutine 84 [running]:
github.com/hashicorp/terraform-provider-aws/internal/service/datazone.(*resourceProject).ImportState(0x1161aea28?, {0x1161aea28, 0x1400224a600}, {{0x1400186fca0?, 0x0?}, {0x0?}}, 0x14001e004e0)
	github.com/hashicorp/terraform-provider-aws/internal/service/datazone/project.go:323 +0x278
github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider.(*wrappedResource).ImportState(0x14004c78380?, {0x1161aea28?, 0x14001174d80?}, {{0x1400186fca0?, 0x110fbf96a?}, {0x2d?}}, 0x14001e004e0)
	github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider/intercept.go:329 +0xb4
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ImportResourceState(0x14000fce960, {0x1161aea28, 0x14001174d80}, 0x14001025ea0, 0x140021e95d0)
	github.com/hashicorp/terraform-plugin-framework@v1.11.0/internal/fwserver/server_importresourcestate.go:136 +0x678
github.com/hashicorp/terraform-plugin-framework/internal/proto5server.(*Server).ImportResourceState(0x14000fce960, {0x1161aea28?, 0x14001174c90?}, 0x14000daea50)
	github.com/hashicorp/terraform-plugin-framework@v1.11.0/internal/proto5server/server_importresourcestate.go:47 +0x248
github.com/hashicorp/terraform-plugin-mux/tf5muxserver.(*muxServer).ImportResourceState(0x14000f9ba40, {0x1161aea28?, 0x14000daeab0?}, 0x14000daea50)
	github.com/hashicorp/terraform-plugin-mux@v0.16.0/tf5muxserver/mux_server_ImportResourceState.go:36 +0x184
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ImportResourceState(0x140010ca320, {0x1161aea28?, 0x14000abd020?}, 0x14000d7cbe0)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:896 +0x1e8
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ImportResourceState_Handler({0x115cddd20, 0x140010ca320}, {0x1161aea28, 0x14000abd020}, 0x14004cbe780, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:536 +0x1c0
google.golang.org/grpc.(*Server).processUnaryRPC(0x14000f2e200, {0x1161aea28, 0x14000abcf60}, {0x116219aa0, 0x140002aa300}, 0x14000a44c60, 0x1400214b350, 0x11f92f010, 0x0)
	google.golang.org/grpc@v1.63.2/server.go:1369 +0xb58
google.golang.org/grpc.(*Server).handleStream(0x14000f2e200, {0x116219aa0, 0x140002aa300}, 0x14000a44c60)
	google.golang.org/grpc@v1.63.2/server.go:1780 +0xb20
google.golang.org/grpc.(*Server).serveStreams.func2.1()
	google.golang.org/grpc@v1.63.2/server.go:1019 +0x8c
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 43
	google.golang.org/grpc@v1.63.2/server.go:1030 +0x13c

Error: The terraform-provider-aws_v5.63.0_x5 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```


### Terraform Configuration Files

import {
  to = aws_datazone_project.cdsvp6st8l82lj
  id = "cdsvp6st8l82lj"
}

### Steps to Reproduce



terraform plan -generate-config-out=resources.out

### Debug Output

_No response_

### Panic Output

Stack trace from the terraform-provider-aws_v5.63.0_x5 plugin:

panic: runtime error: index out of range [1] with length 1

goroutine 81 [running]:
github.com/hashicorp/terraform-provider-aws/internal/service/datazone.(*resourceProject).ImportState(0x1161aea28?, {0x1161aea28, 0x140021716b0}, {{0x1400186f230?, 0x0?}, {0x0?}}, 0x14001226180)
	github.com/hashicorp/terraform-provider-aws/internal/service/datazone/project.go:323 +0x278
github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider.(*wrappedResource).ImportState(0x14002dbc780?, {0x1161aea28?, 0x140009eb9e0?}, {{0x1400186f230?, 0x110fbf96a?}, {0x2d?}}, 0x14001226180)
	github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider/intercept.go:329 +0xb4
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ImportResourceState(0x14000fce960, {0x1161aea28, 0x140009eb9e0}, 0x14000fe9730, 0x140021d95d0)
	github.com/hashicorp/terraform-plugin-framework@v1.11.0/internal/fwserver/server_importresourcestate.go:136 +0x678
github.com/hashicorp/terraform-plugin-framework/internal/proto5server.(*Server).ImportResourceState(0x14000fce960, {0x1161aea28?, 0x140009eb8f0?}, 0x14000abc9f0)
	github.com/hashicorp/terraform-plugin-framework@v1.11.0/internal/proto5server/server_importresourcestate.go:47 +0x248
github.com/hashicorp/terraform-plugin-mux/tf5muxserver.(*muxServer).ImportResourceState(0x14000f9ba40, {0x1161aea28?, 0x14000abca50?}, 0x14000abc9f0)
	github.com/hashicorp/terraform-plugin-mux@v0.16.0/tf5muxserver/mux_server_ImportResourceState.go:36 +0x184
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ImportResourceState(0x140010ca320, {0x1161aea28?, 0x14000a51ec0?}, 0x14000d7c190)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:896 +0x1e8
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ImportResourceState_Handler({0x115cddd20, 0x140010ca320}, {0x1161aea28, 0x14000a51ec0}, 0x14004cbe000, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:536 +0x1c0
google.golang.org/grpc.(*Server).processUnaryRPC(0x14000f2e200, {0x1161aea28, 0x14000a50ae0}, {0x116219aa0, 0x140002aa300}, 0x140009b8000, 0x1400214b350, 0x11f92f010, 0x0)
	google.golang.org/grpc@v1.63.2/server.go:1369 +0xb58
google.golang.org/grpc.(*Server).handleStream(0x14000f2e200, {0x116219aa0, 0x140002aa300}, 0x140009b8000)
	google.golang.org/grpc@v1.63.2/server.go:1780 +0xb20
google.golang.org/grpc.(*Server).serveStreams.func2.1()
	google.golang.org/grpc@v1.63.2/server.go:1019 +0x8c
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 43
	google.golang.org/grpc@v1.63.2/server.go:1030 +0x13c
panic: runtime error: index out of range [1] with length 1

goroutine 84 [running]:
github.com/hashicorp/terraform-provider-aws/internal/service/datazone.(*resourceProject).ImportState(0x1161aea28?, {0x1161aea28, 0x1400224a600}, {{0x1400186fca0?, 0x0?}, {0x0?}}, 0x14001e004e0)
	github.com/hashicorp/terraform-provider-aws/internal/service/datazone/project.go:323 +0x278
github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider.(*wrappedResource).ImportState(0x14004c78380?, {0x1161aea28?, 0x14001174d80?}, {{0x1400186fca0?, 0x110fbf96a?}, {0x2d?}}, 0x14001e004e0)
	github.com/hashicorp/terraform-provider-aws/internal/provider/fwprovider/intercept.go:329 +0xb4
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ImportResourceState(0x14000fce960, {0x1161aea28, 0x14001174d80}, 0x14001025ea0, 0x140021e95d0)
	github.com/hashicorp/terraform-plugin-framework@v1.11.0/internal/fwserver/server_importresourcestate.go:136 +0x678
github.com/hashicorp/terraform-plugin-framework/internal/proto5server.(*Server).ImportResourceState(0x14000fce960, {0x1161aea28?, 0x14001174c90?}, 0x14000daea50)
	github.com/hashicorp/terraform-plugin-framework@v1.11.0/internal/proto5server/server_importresourcestate.go:47 +0x248
github.com/hashicorp/terraform-plugin-mux/tf5muxserver.(*muxServer).ImportResourceState(0x14000f9ba40, {0x1161aea28?, 0x14000daeab0?}, 0x14000daea50)
	github.com/hashicorp/terraform-plugin-mux@v0.16.0/tf5muxserver/mux_server_ImportResourceState.go:36 +0x184
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ImportResourceState(0x140010ca320, {0x1161aea28?, 0x14000abd020?}, 0x14000d7cbe0)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/tf5server/server.go:896 +0x1e8
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ImportResourceState_Handler({0x115cddd20, 0x140010ca320}, {0x1161aea28, 0x14000abd020}, 0x14004cbe780, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.23.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:536 +0x1c0
google.golang.org/grpc.(*Server).processUnaryRPC(0x14000f2e200, {0x1161aea28, 0x14000abcf60}, {0x116219aa0, 0x140002aa300}, 0x14000a44c60, 0x1400214b350, 0x11f92f010, 0x0)
	google.golang.org/grpc@v1.63.2/server.go:1369 +0xb58
google.golang.org/grpc.(*Server).handleStream(0x14000f2e200, {0x116219aa0, 0x140002aa300}, 0x14000a44c60)
	google.golang.org/grpc@v1.63.2/server.go:1780 +0xb20
google.golang.org/grpc.(*Server).serveStreams.func2.1()
	google.golang.org/grpc@v1.63.2/server.go:1019 +0x8c
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 43
	google.golang.org/grpc@v1.63.2/server.go:1030 +0x13c

Error: The terraform-provider-aws_v5.63.0_x5 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.

### Important Factoids

_No response_

### References

_No response_

### Would you like to implement a fix?

None

--- a/.changelog/38924.txt
+++ b/.changelog/38924.txt
@@ -1,0 +1,3 @@
+```release-note:bug
+resource/aws_datazone_project: Properly surface import `id` parsing errors
+```

--- a/internal/service/datazone/project.go
+++ b/internal/service/datazone/project.go
@@ -318,7 +318,9 @@ func (r *resourceProject) ImportState(ctx context.Context, req resource.ImportSt
 
 	if len(parts) != 2 {
 		resp.Diagnostics.AddError("Resource Import Invalid ID", fmt.Sprintf(`Unexpected format for import ID (%s), use: "DomainIdentifier:Id"`, req.ID))
+		return
 	}
+
 	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, path.Root("domain_identifier"), parts[0])...)
 	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, path.Root(names.AttrID), parts[1])...)
 }
@@ -326,7 +328,7 @@ func (r *resourceProject) ImportState(ctx context.Context, req resource.ImportSt
 func waitProjectCreated(ctx context.Context, conn *datazone.Client, domain string, identifier string, timeout time.Duration) (*datazone.GetProjectOutput, error) {
 	stateConf := &retry.StateChangeConf{
 		Pending:                   []string{},
-		Target:                    enum.Slice[awstypes.ProjectStatus](awstypes.ProjectStatusActive),
+		Target:                    enum.Slice(awstypes.ProjectStatusActive),
 		Refresh:                   statusProject(ctx, conn, domain, identifier),
 		Timeout:                   timeout,
 		NotFoundChecks:            40,
@@ -343,7 +345,7 @@ func waitProjectCreated(ctx context.Context, conn *datazone.Client, domain strin
 
 func waitProjectDeleted(ctx context.Context, conn *datazone.Client, domain string, identifier string, timeout time.Duration) (*datazone.GetProjectOutput, error) {
 	stateConf := &retry.StateChangeConf{
-		Pending: enum.Slice[awstypes.ProjectStatus](awstypes.ProjectStatusDeleting, awstypes.ProjectStatusActive),
+		Pending: enum.Slice(awstypes.ProjectStatusDeleting, awstypes.ProjectStatusActive),
 		Target:  []string{},
 		Refresh: statusProject(ctx, conn, domain, identifier),
 		Timeout: timeout,

--- a/website/docs/r/datazone_project.html.markdown
+++ b/website/docs/r/datazone_project.html.markdown
@@ -34,14 +34,14 @@ resource "aws_datazone_project" "test" {
 
 The following arguments are required:
 
-* `domain_identifier` - (Required) Identifier of domain which the project is part of. Must follow the regex of ^dzd[-_][a-zA-Z0-9_-]{1,36}$.
-* `name` - (Required) Name of the project. Must follow the regex of ^[\w -]+$. and have a length of at most 64.
+* `domain_identifier` - (Required) Identifier of domain which the project is part of. Must follow the regex of `^dzd[-_][a-zA-Z0-9_-]{1,36}$`.
+* `name` - (Required) Name of the project. Must follow the regex of `^[\w -]+$`. and have a length of at most 64.
 
 The following arguments are optional:
 
 * `skip_deletion_check` - (Optional) Optional flag to delete all child entities within the project.
 * `description` - (Optional) Description of project.
-* `glossary_terms` - (Optional) List of glossary terms that can be used in the project. The list cannot be empty or include over 20 values. Each value must follow the regex of [a-zA-Z0-9_-]{1,36}$.
+* `glossary_terms` - (Optional) List of glossary terms that can be used in the project. The list cannot be empty or include over 20 values. Each value must follow the regex of `[a-zA-Z0-9_-]{1,36}$`.
 
 ## Attribute Reference
 
@@ -56,7 +56,7 @@ This resource exports the following attributes in addition to the arguments abov
 * `failure_reasons` - List of error messages if operation cannot be completed.
 * `glossary_terms` - Business glossary terms that can be used in the project.
 * `last_updated_at` - Timestamp of when the project was last updated.
-* `project_status` -  Enum that conveys state of project. Can be ACTIVE, DELETING, or DELETE_FAILED.
+* `project_status` -  Enum that conveys state of project. Can be `ACTIVE`, `DELETING`, or `DELETE_FAILED`.
 
 ## Timeouts
 
@@ -67,17 +67,17 @@ This resource exports the following attributes in addition to the arguments abov
 
 ## Import
 
-In Terraform v1.5.0 and later, use an [`import` block](https://developer.hashicorp.com/terraform/language/import) to import DataZone Project using the `id`. For example:
+In Terraform v1.5.0 and later, use an [`import` block](https://developer.hashicorp.com/terraform/language/import) to import DataZone Project using a colon-delimited string combining `domain_id` and `id`. For example:
 
 ```terraform
 import {
   to = aws_datazone_project.example
-  id = "projectid123"
+  id = "domain-1234:project-1234"
 }
 ```
 
-Using `terraform import`, import DataZone Project using the `id`. For example:
+Using `terraform import`, import DataZone Project using a colon-delimited string combining `domain_id` and `id`. For example:
 
 ```console
-% terraform import aws_datazone_project.example projectid123
+% terraform import aws_datazone_project.example domain-1234:project-1234
 ```

